pkg/kubelet/cm/topologymanager/numa_info.go

```diff
@@ -47,7 +47,7 @@ func NewNUMAInfo(topology []cadvisorapi.Node) (*NUMAInfo, error) {
 
 func newNUMAInfo(topology []cadvisorapi.Node, sysFs *NUMASysFs) (*NUMAInfo, error) {
 	var numaNodes []int
-	distances := make([][]uint64, len(topology))
+	distances := map[int][]uint64{}
```

Comment:
Ugh, I should have caught this in the initial review

Reply:
I thought NUMA nodes are always ordered from 0 but it seems that is not true 😞